.prettierignore

```diff
@@ -2,3 +2,4 @@
 
 /dist
 /coverage
+/packages/libs/api/sdk/src/remote/graphql/index.ts
```

Comment:
The generated file doesn't comply with our prettier rules. We could add a hook to fix it but [according to them](https://www.the-guild.dev/graphql/codegen/docs/integrations/prettier)

`Before adding a hook, consider the alternative of having codegen ignored in your linting. Codegen files should not be edited manually and formatting them slows down your codegen considerably. Differences can be measured in several seconds for every run on big projects.
`

I would love to have a `.prettierignore` inside the package folder instead of making it in this global file, however I didn't manage to do it and seems like even [NX repo uses the global](https://github.com/nrwl/nx/blob/master/.prettierignore) `.prettierignore` as well :/

Reply:
Makes sense, thanks for the explanation!